rav1e_js/src/encoder_config.rs

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) 2020, The rav1e contributors. All rights reserved
+//
+// This source code is subject to the terms of the BSD 2 Clause License and
+// the Alliance for Open Media Patent License 1.0. If the BSD 2 Clause License
+// was not distributed with this source code in the LICENSE file, you can
+// obtain it at www.aomedia.org/license/software. If the Alliance for Open
+// Media Patent License 1.0 was not distributed with this source code in the
+// PATENTS file, you can obtain it at www.aomedia.org/license/patent.
+
+#![allow(non_snake_case)]
+
+use rav1e::prelude::{
+  ChromaSamplePosition, ChromaSampling, ColorDescription, ContentLight,
```

Comment:
`ColorDescription` and `MasteringDisplay` seems to be unused import

Reply:
Yes. These are needed for `.setColorDescription(...)` and `.setMasteringDisplay(...)`, which are currently commented out, because these types require some further work to be supported for wasm-bindgen.

I would actually just leave it like that, but if you prefer I can also remove it.

Reply:
Okay, in that case, we can include it when we are going to use it, I think not including and skipping warning is a better option

Reply:
Done